### ISSUE TITLE
Book 05: Sentence 040

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#15]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/7000a6a5-e93a-4c61-aba8-977f4352d360)

```
# sent_id = Book_05_-_The_Omens.txt:40
# text = Ic nauj capitulo, vncan mjtoa: in tetzaujtl injc motetzaujaia, in jquac teculutl chocaia.
# text[norm] = Ic nahui capítulo, oncan mihtoa: in tetzahuitl inic motetzahuiaya, in ihcuac tecolotl chocaya.
# text[orig] = I·c·nauj·capitulo·,·vncan·mj¶toa·:·intetzaujtl·injc·motetzauja¶ia·,·inj·quac·teculutl·chocaia·.
# text[osp] = Capitulo quarto, del mal aguero: que tomauan del canto del buho ave.
# text[spa] = Cuarto capítulo, donde se habla del agüero, que tomaron por agüero, cuando el tecolote ulula.
# references = A&D/168:1
1	Ic	ic	ADV	_	_	2	_	_	_
2	nauj	nahui	NUM	_	_	3	_	_	_
3	capitulo	capítulo	NOUN	_	State=Abs	6	_	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	vncan	oncan	ADV	_	_	6	_	_	_
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
7	:	:	PUNCT	_	_	9	punct	_	_
8	in	in	DET|SCONJ	_	_	9	_	_	_
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	_	_	_
10	injc	inic	SCONJ	_	_	11	_	_	_
11	motetzaujaia	tetzahuia	VERB	_	_	9	_	_	_
12	,	,	PUNCT	_	_	16	punct	_	_
13	in	in	DET|SCONJ	_	_	16	_	_	_
14	jquac	ihcuac	SCONJ	_	_	16	_	_	_
15	teculutl	tecolotl	NOUN	_	State=Abs	16	_	_	_
16	chocaia	choca	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	11	_	_	_
17	.	.	PUNCT	_	_	6	punct	_	_
```

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/b2ec8bd1-026c-416b-a8e3-61e83e10e1b0)

Updated according to decisions in Sentence 001 (#14) and 028 (#4).

Remaining questions:
- Status of *in* with *tetzahuitl* (see also other issues ad nauseum)
- Status of *inic* and the clause it introduces, does it modify *tetzahuitl* as `acl` or *mihtoa* as `advcl`?

**Comment by Lguyogiro** [comment]
Working on the rel from _tetzahuitl_ / _motetzahuiaya_ and _chocaya_ in #34

**Comment by ftyers** [comment]
According to #34, this is what it should be:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/a512071d-738c-446a-8946-20f70a16719e)

Although I'm not super convinced...

**Comment by Lguyogiro** [comment]
That's the only part that is weird to me, having one of the  parts of a `conj` relationship introduced by a subordinator that is different than the other... whereas here, I kind of want to do `acl(tetzahuitl, motetzahuiaya)` ? lol but we did not do that in #34 

So either the above is good or we should change 34 to have this acl rel ("it was a portent that VERBs").

I'm happy with either one.

**Comment by ftyers** [comment]
> That's the only part that is weird to me, having one of the parts of a `conj` relationship introduced by a subordinator that is different than the other... whereas here, I kind of want to do `acl(tetzahuitl, motetzahuiaya)` ? lol but we did not do that in #34
>

Well, it's not completely different right? *inic* = *in ic*, we could think of it like `mark(..., in)`, `advmod(..., ic)`

**Comment by Lguyogiro** [comment]
Yeah good point. LGTM

**Comment by ftyers** [comment]
Merged in 30e3597.
